make_obsparams.py

```diff
@@ -72,14 +118,11 @@ def _make_h4c_obsparam(freq_range, freqs, sky_model, chunks):
                 "polarization_array": [-5, -7, -8, -6],
             }
 
-            obsparams_file = f"{obsparams_dir}/fch{fch:04d}_chunk{ch}.yaml"
 
             with open(obsparams_file, "w") as stream:
                 yaml.dump(obsparams, stream, default_flow_style=False, sort_keys=False)
 
-
-"""===CLI==="""
-
+    return layout_file
 
 @click.group(context_settings=CONTEXT_SETTINGS)
 def cli():
```

Comment:
Now that any layout can be supported through the `--layout flag`, I think we can get rid of this `cli` function and the cli grouping (L127), which was originally implemented to allow a separate command for each layout.

Reply:
Good point. Done.